posthog/queries/stickiness/stickiness_event_query.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, Dict, Tuple
+
+from posthog.constants import TREND_FILTER_TYPE_ACTIONS, PropertyOperatorType
+from posthog.models import Entity
+from posthog.models.action.util import format_action_filter
+from posthog.models.filters.stickiness_filter import StickinessFilter
+from posthog.models.utils import PersonPropertiesMode
+from posthog.queries.event_query import EventQuery
+from posthog.queries.util import get_trunc_func_ch
+
+
+def stickiness_event_factory(event_query_class: EventQuery):
```

Comment:
This feels unideal. But posthog vs ee query classes need to inherit different base classes